knip.json

```diff
@@ -0,0 +1,40 @@
+{
+  "$schema": "https://unpkg.com/knip@3/schema.json",
+  "workspaces": {
+    "apps/backend": {
+      "entry": [
+        "src/processes/proc/*.ts",
+        "src/build/bin/*.ts",
+        "src/database/testing/index.ts",
+        "src/graphql/definitions/*.ts"
+      ],
+      "ignore": ["db", "knexfile.js", "src/graphql/__generated__"],
+      "ignoreDependencies": ["pg"],
+      "paths": {
+        "@/*": ["./src/*"]
```

Comment:
When I remove this I get the same results. I think this `paths` config can be removed, because `tsconfig.json` has it (and Knip uses that as well).